From ddba51c0563149a0bfa17c8492d8776c951aab71 Mon Sep 17 00:00:00 2001
From: yvanliang <yvan6liang@outlook.com>
Date: Fri, 9 Oct 2026 15:43:40 +0800
Subject: [PATCH 11/23] GSI Android 17 compatibility:
 0011-Add-vendor-firmware_mnt.-Moto-Razr-2019-look-for-fir.patch

Change-Id: I5c2493f34684815d244ec8421081cf63fb5bd87a
---
 rootdir/ueventd.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 1da9707..8d92adb 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -1,7 +1,7 @@
 import /vendor/etc/ueventd.rc
 import /odm/etc/ueventd.rc
 
-firmware_directories /etc/firmware/ /odm/firmware/ /vendor/firmware/ /firmware/image/
+firmware_directories /etc/firmware/ /odm/firmware/ /vendor/firmware/ /firmware/image/ /vendor/firmware_mnt/image/
 uevent_socket_rcvbuf_size 16M
 
 subsystem graphics
